README.md

```diff
@@ -2,474 +2,36 @@
 
 # kodiak [![CircleCI](https://circleci.com/gh/chdsbd/kodiak.svg?style=svg&circle-token=4879604a0cca6fa815c4d22936350f5bdf455905)](https://circleci.com/gh/chdsbd/kodiak)
 
-A bot to automatically merge GitHub PRs
+> A GitHub bot to automatically update and merge GitHub PRs
 
-[![install](https://3c7446e0-cd7f-4e98-a123-1875fcbf3182.s3.amazonaws.com/button-small.svg?v=123)](https://github.com/marketplace/kodiakhq#pricing-and-setup)
+Kodiak automates your GitHub workflow:
 
-## Why?
+- Simple & Configurable — a simple configuration file with smart defaults will get you started in minutes
+- Update — update your branches automatically
+- Merge — use Kodiak's simple configuration with GitHub Branch Protection to merge pull requests automatically
+- Delete — remove your old branches automatically
 
-Enabling the "require branches be up to date" feature on GitHub repositories is
-great because, when coupled with CI, _master will always be green_.
+Configure Kodiak to automate your workflow in minutes!
 
-However, as the number of collaborators on a GitHub repo increases, a
-repetitive behavior emerges where contributors are updating their branches
-manually hoping to merge their branch before others.
+## Installation
 
-Kodiak fixes this wasteful behavior by _automatically updating
-and merging branches_. Contributors simply mark their
-PR with a (configurable) label to indicate the PR is ready to merge and Kodiak
-will do the rest; handling branch updates and merging using the _minimal
-number of branch updates_ to land the code on master.
+Kodiak is available through the GitHub Marketplace.
 
-This means that contributors don't have to worry about keeping their PRs up
-to date with the latest on master or even pressing the merge button. Kodiak
-does this for them! 🎉
+[![install](https://3c7446e0-cd7f-4e98-a123-1875fcbf3182.s3.amazonaws.com/button-small.svg?v=123)](https://github.com/marketplace/kodiakhq)
 
-### Minimal updates
+If you'd rather run Kodiak yourself, check out the [self hosting page]() in our docs.
 
-Kodiak _efficiently updates pull requests_ by only updating a PR when it's ready to merge. This
-_prevents spurious CI jobs_ from being created as they would if all PRs were
-updated when their targets were updated.
+## [Documentation](https://kodiakhq.com)
 
-## How does it work?
+Helpful Links:
 
-1.  Kodiak receives a webhook event from GitHub and adds it to a per-installation queue for processing
-2.  Kodiak processes these webhook events and extracts the associated pull
-    requests for further processing
+- [Getting Started](https://kodiakhq.com/docs/quickstart)
+- [Why and How](https://kodiakhq.com/docs/why-and-how)
+- [Troubleshooting](https://kodiakhq.com/docs/troubleshooting)
+- [Help](https://kodiakhq.com/help)
```

Comment:
I think it would be useful to have a link to a configuration reference here. Maybe a link to the comparison without other solutions, but that's not too big of a concern for me.